### PR TITLE
init creates the default component files #371

### PR DESCRIPTION
**Pull request by vinayada1**
# Description

dapr init creates default components folder with files inside.

## #371 

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [* ] Code compiles correctly
* [ *] Created/updated tests
* [ *] Extended the documentation

### CODE REVIEW

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -197,7 +206,7 @@ func getDownloadDest(installLocation string) (string, error) {
 
 // installLocation is not used, but it is present because it's required to fit the initSteps func above.
 // If the number of args increases more, we may consider passing in a struct instead of individual args.
-func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
+func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string, _ string) {
```
Comment (pruthvidhodda) [inline pkg/standalone/standalone.go:209]: Do we need to run redis container when redisHost is given - we won't be using this container anyways right?

File: README.md [hunk 2]
```diff
@@ -175,7 +181,8 @@ $ dapr run --app-id nodeapp --app-port 3000 --grpc-port 50002 node app.js
 Example of launching Dapr within a specific Docker network:
 
 ```bash
-$ dapr run --app-id nodeapp --redis-host dapr_redis --placement-host dapr_placement node app.js
+$ dapr init --redis-host dapr_redis
```
Comment (mukundansundar) [inline README.md:184]: shouldn't this have a `--network` flag?

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (mukundansundar) [reply pkg/standalone/standalone.go:209]: 👍

File: README.md (hunk 2 above)
Comment (vinayada1) [reply README.md:184]: init already has a --network flag

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (vinayada1) [reply pkg/standalone/standalone.go:209]: You mean if the redis-host != localhost? But what if the user specifies it as 127.0.0.1 or specifies the IP of the machine itself? redis-host != localhost might not be good enough to decide that

Comment (vinayada1) [reply pkg/standalone/standalone.go]: As discussed offline, if the user specifies a redis-host, the user will need to make sure it is running (even if the user has specified the same machine)

File: README.md (hunk 2 above)
Comment (vinayada1) [reply README.md:184]: Added

File: pkg/standalone/run_test.go [hunk 3]
```diff
@@ -1,133 +1,131 @@
...
+
+		assert.Contains(t, output.DaprCMD.Args[0], "daprd")
+		assertArgument(t, "app-id", "MyID", output.DaprCMD.Args)
+		assertArgument(t, "dapr-http-port", "8000", output.DaprCMD.Args)
+		assertArgument(t, "dapr-grpc-port", "50001", output.DaprCMD.Args)
+		assertArgument(t, "log-level", "WARN", output.DaprCMD.Args)
+		assertArgument(t, "max-concurrency", "-1", output.DaprCMD.Args)
+		assertArgument(t, "protocol", "http", output.DaprCMD.Args)
+		assertArgument(t, "app-port", "3000", output.DaprCMD.Args)
+		assertArgument(t, "components-path", GetDefaultComponentsFolder(), output.DaprCMD.Args)
```
Comment (vinayada1) [inline pkg/standalone/run_test.go:82]: Had control chars....this file is now same as dapr/master except removed RedisHost: "localhost" while calling t.Run()

**Comment by yaron2** [comment]
> /ok-to-test

This command is used to run e2e tests in the `dapr/dapr` repo :)

**Comment by yaron2** [comment]
build tests failing due to linter: https://github.com/dapr/cli/pull/373/checks?check_run_id=773484056

File: README.md [hunk 4]
```diff
@@ -175,7 +181,8 @@ $ dapr run --app-id nodeapp --app-port 3000 --grpc-port 50002 node app.js
 Example of launching Dapr within a specific Docker network:
 
 ```bash
-$ dapr run --app-id nodeapp --redis-host dapr_redis --placement-host dapr_placement node app.js
+$ dapr init --redis-host dapr_redis --network dapr-network
```
Comment (amanbha) [inline README.md:184]: What s dapr_redis here?

Comment (vinayada1) [reply README.md:184]: Name of the redis host. This is same as the previous code except moved it from dapr run to dapr init

**Pull request merged**